src/views/guidelines/guidelines.scss

```diff
@@ -0,0 +1,16 @@
+@import "../../colors";
+@import "../../typography";
+
+#view {
+    .guidelines-footer {
+        text-align: center;
```

Comment:
thanks :). I think the linter wants these to be switched though – `margin-top` above `text-align`